flink-runtime/src/test/java/org/apache/flink/runtime/rest/MultipartUploadExtension.java

```diff
@@ -63,18 +64,18 @@
 import java.util.List;
 import java.util.Objects;
 import java.util.concurrent.CompletableFuture;
+import java.util.function.Supplier;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
```

Comment:
i guess it could be replaced with assertj
WDYT?

Reply:
Yes, good catch! 👍